.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,21 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      env:
+       HUB_LOGIN: "${{ secrets.HUB_LOGIN }}"
+    - run: |
+        IMAGE_NAME="universityofwarwick/sso-stub:latest"
+        ./buildDocker.sh
+        docker image tag sso-stub:latest ${IMAGE_NAME}
+        echo "${HUB_LOGIN}" | docker login -u adamlwilliams --password-stdin
```

Comment:
Should we be logging in as your account here?

Reply:
Unless you have an alternative, yeah.. (it's not used for anything other than this)

It's publishing to an org but there aren't org-specific tokens.

Reply:
I'd love to use GitHub's but it's got a bunch of its own problems

- They have an old per-repo registry: Can't be used to pull images unauthed
- The new org-based container registries: Can't have images pushed into them using the GitHub Actions access token, personal access tokens require access to all private repositories, no way to scope down to just this